CLI/Analyze.cpp

```diff
@@ -6,6 +6,9 @@
 #include "Luau/TypeAttach.h"
 #include "Luau/Transpiler.h"
 
+#include <iterator>
+#include <iostream>
```

Comment:
Please use stdio.h instead - we try to use it exclusively to avoid paying the cost in terms of the binary size.

Reply:
Done. Good point, I have changed the code to only use `fgets` to read from stdin